src/components/Pages/Form/Form.tsx

```diff
@@ -143,7 +144,7 @@ const HabitForm = () => {
               }
             />
             <FormHelperText id="rewards-helper-text">
-              {formik.touched.rewards && formik.errors.rewards}
+              {formik.touched.rewards && (formik.errors.rewards as string)}
```

Comment:
Do you need to type case here?

Reply:
I think so

Reply:
So what is the default type here `formik.errors.rewards`? `undefined` | string?